configs/rodent.yaml

```diff
@@ -1,5 +1,13 @@
 # Frames per clip for transform.
-N_FRAMES_PER_CLIP: 360
+N_FRAMES_PER_CLIP: 250
+
+# Tolerance for the optimizations of the full model, limb, and root. 
```

Comment:
If these tolerances are unused, please add a comment, and add a reference to the PR to fix it.